Testscripts/Linux/WALA-VERIFY-FIREWALL-STATUS.py

```diff
@@ -17,8 +17,9 @@
 EXECUTION_USER = "root"
 WIRESERVER_ENDPOINT_FILE = '/var/lib/waagent/WireServerEndpoint'
 VERSIONS_PATH = '/?comp=versions'
-AGENT_CONFIG_FILE = '/etc/waagent.conf'
 OS_ENABLE_FIREWALL_RX = r'OS.EnableFirewall\s*=\s*(\S+)'
+AGENT_CONFIG_FILE = Run("find / -name waagent.conf")
```

Comment:
Why do we need this change? waagent.conf location is static.

Reply:
it has different path in coreos

Reply:
Should we open and ask waagent team fix the path?I can merge this way but we have a few other scripts with the old way.

Reply:
I see it's end-of-life, June 2020. No need to change then.